### PR TITLE
Mesos/Marathon health checks conflict

### PR DESCRIPTION
**Pull request by kensipe**
Mesos/Marathon health checks conflict

Summary:
Added details regarding the lack of support combining marathon and mesos health checks
https://jira.mesosphere.com/browse/MARATHON-7611

JIRA issues: MARATHON-7611

### CODE REVIEW

File: docs/docs/health-checks.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Mesos-level health checks (`MESOS_HTTP`, `MESOS_HTTPS`, `MESOS_TCP`, and `COMMAN
 
 - Mesos-level health checks require tasks to listen on the container's loopback interface in addition to whatever interface they require. If you run a service in production, you will want to make sure that the users can reach it.
 
+- Marathon currently does NOT support the combination of mesos and marathon level health checks.
```
Comment (ichernetsky) [inline docs/docs/health-checks.md]: nit: `Mesos and Marathon`.

Comment (janisz) [reply docs/docs/health-checks.md]: I'd prefer to enforce it in `AppDefinition` validation rather than just saying in the documentation

Comment (kensipe) [reply docs/docs/health-checks.md]: hey... @janisz !!!  good point.  I will add a JIRA to that effect!

File: docs/docs/health-checks.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ Mesos-level health checks (`MESOS_HTTP`, `MESOS_HTTPS`, `MESOS_TCP`, and `COMMAN
 
 - Mesos-level health checks require tasks to listen on the container's loopback interface in addition to whatever interface they require. If you run a service in production, you will want to make sure that the users can reach it.
 
+- Marathon currently does NOT support the combination of Mesos and Marathon level health checks.
```
Comment (timcharper) [inline docs/docs/health-checks.md]: Do you know if we reject such combinations?

File: docs/docs/health-checks.md (hunk 1 above)
Comment (kensipe) [reply docs/docs/health-checks.md]: https://jira.mesosphere.com/browse/MARATHON-8369

File: docs/docs/health-checks.md (hunk 2 above)
Comment (kensipe) [reply docs/docs/health-checks.md]: https://jira.mesosphere.com/browse/MARATHON-8369

good catch.. this JIRA is to "reject" this combination.   Currently it is not rejected and it doesn't work properly if used.

**Pull request merged**